docs/source/roles.md

```diff
@@ -51,22 +62,34 @@ The software engineers responsible for developing and maintaining TRE software,
 
 * - Role name
   - Role description
-* -
-  -
+* - TRE Software Developer
+  - People responsible for building the software infrastructure that can be used as a TRE.
 ```
 
+(governance_roles)=
+
 ## TRE governance roles
 
 Roles that uphold the governance of TREs.
 Such governance responsibilities typically involve establishing policies and procedures to ensure the responsible use of data, protecting the privacy and confidentiality of research participants, and promoting transparency and accountability in research activities.
-Typical roles might include data custodians, ethicists, an independent board or a lay panel.
 
 ```{list-table}
 :header-rows: 1
 :name: tab-tre-role-governance
 
 * - Role name
   - Role description
-* -
+* - Quality Manager
+  - People responsible for ensuring the TRE is operating correctly and all processes are working as intended, and being followed by other roles.
+* - Top Management
+  - This is an ISO term for the most senior governance who own the risks, make decisions and allocate resources.
```

Comment:
```suggestion
  - This is an ISO 9001 term for the most senior governance who own the risks, make decisions and allocate resources.
```
Do we need to link to the relevant part of ISO 9001?

Reply:
@machintim would you know where this is?